chatsky/pipeline/service/group.py

```diff
@@ -61,26 +52,41 @@ class ServiceGroup(PipelineComponent, extra="forbid", arbitrary_types_allowed=Tr
             ServiceGroup,
         ]
     ]
+    """
+    A `ServiceGroup` object, that will be added to the group.
+    """
+    # Inherited fields repeated. Don't delete these, they're needed for documentation!
+    before_handler: BeforeHandler = Field(default_factory=BeforeHandler)
+    after_handler: AfterHandler = Field(default_factory=AfterHandler)
+    timeout: Optional[float] = None
+    requested_async_flag: Optional[bool] = None
+    start_condition: StartConditionCheckerFunction = Field(default=always_start_condition)
+    name: Optional[str] = None
+    path: Optional[str] = None
```

Comment:
Did `inherited-members` not work as expected?

Reply:
Yes, it also inherits all methods of parent class, which are not needed, If I'm not wrong. Also, all inherited fields and methods are added at the end of the previous doc file, you could say. Meaning it didn't solve the issue at all and made it worse.
Using 'member-order': "groupwise" doesn't really help, because then methods go before fields for some reason. Though, within groups members are alphabetical, removing the distinction between differing classes. If the order can be tweaked, it would be fixed, but then `Actor` would look bad, because it doesn't need to inherit all of the fields in the docs. And inheriting all of the methods also seems unnecessary.

I think this is an okay solution for now. Either docs look good, or code looks good. This code is lacking in code-style, but the docs become broken/unreadable otherwise.
Well, maybe "broken docs" is just "not finished", really. I think there were extensions for Sphinx with Pydantic? Or extensions for sphinx-autodoc, those exist. This could take a long time, though.